.docs/user-guide/docker-plugins.md

```diff
@@ -147,11 +247,38 @@ Environment Variable | Description | Default | Required
 REX-Ray ships with plug-ins for Google Compute Engine (GCE) as well.
 
 ### GCE Persistent Disk
+The GCEPD plug-in can be installed with the following command:
 
-## Microsoft
-Microsoft Azure plug-ins are included with REX-Ray as well.
+```bash
+$ docker plugin install rexray/gcepd \
+  GCEPD_TAG=rexray
+```
+
+#### Requirements
+The GCEPD plug-in requires that GCE compute instance has Read/Write Cloud API
+access to the Compute Engine and Storage services.
+
+#### Privileges
+The GCEPD plug-in requires the following privileges:
+
+Type | Value
+-----|------
+network | `host`
+mount | `/dev`
+allow-all-devices | `true`
+capabilities | `CAP_SYS_ADMIN`
 
-### Azure Unmanaged Disk
+#### Configuration
+The following environment variables can be used to configure the GCEPD
+plug-in:
+
+Environment Variable | Description | Default | Required
+---------------------|-------------|---------|---------
+`GCEPD_DEFAULTDISKTYPE` | The default disk type to consume | `pd-ssd` |
+`GCEPD_PROJECTID` | GCE Project ID | `true` |
+`GCEPD_REGION` | GCE Region | `false` |
```

Comment:
This var does not exist.  Only `zone` `disktype` and `tag`.  So just delete this and the one before it.